api/apps/api/src/modules/scenarios/locks/lock.service.spec.ts

```diff
@@ -64,13 +72,17 @@ async function getFixtures() {
         provide: getRepositoryToken(ScenarioLockEntity),
         useValue: {
           manager: {
-            transaction: (fn: (em: Partial<EntityManager>) => Promise<void>) =>
+            transaction: (fn: (em: any) => Promise<void>) =>
```

Comment:
I'm sure you have good reasons for this (including preserving your sanity against TS's type checker 😬), but switching to `any` always rings loud alerts in my brain both when I do it myself and when I do CRs... can we keep this more strict?